src/pretix/base/customersso/oidc.py

```diff
@@ -154,6 +165,11 @@ def oidc_authorize_url(provider, state, redirect_uri):
         'state': state,
         'redirect_uri': redirect_uri,
     }
+
+    if "query_parameters" in provider.configuration and provider.configuration["query_parameters"]:
+        for qp in provider.configuration["query_parameters"].split("&"):
+            params[qp.split("=")[0]] = qp.split("=")[1]
+
```

Comment:
Again, I'd recommend using parse_qsl. This also allows the user to actually use correctly escaped values. Otherwise, they would be double-escaped here.
```suggestion

    if "query_parameters" in provider.configuration and provider.configuration["query_parameters"]:
        params.update(urllib.parse.parse_qsl(provider.configuration["query_parameters"]))

```